minions/async/src/test/java/io/github/microcks/minion/async/AsyncAPITestManagerIT.java

```diff
@@ -166,4 +166,94 @@ private static void sendTextMessagesOnTopic(int number) {
       producer.flush();
       producer.close();
    }
+
+   @Test
+   void testKafkaSendMessageTestSuccess() throws Exception {
```

Comment:
I don't get the naming and why it's a `KafkaSendMessage` test? It looks super similar to the `testKafkaAsyncAPITestSuccess()` above. Couldn't have we done this with a `ParametrizedTest`?